packages/python/plotly/plotly/tests/test_optional/test_utils/test_utils.py

```diff
@@ -372,38 +372,6 @@ def test_invalid_encode_exception(self):
         with self.assertRaises(TypeError):
             _json.dumps({"a": {1}}, cls=utils.PlotlyJSONEncoder)
 
-    def test_fast_track_finite_arrays(self):
```

Comment:
@marthacryan 
Could we revert this change now or we should drop the test?

Reply:
I removed this because it was failing. Presumably it's just faster now?